static/programs/awsx-root-directory-go/go.mod

```diff
@@ -0,0 +1,8 @@
+module awsx-root-directory-go
+
+go 1.21.12
```

Comment:
Should this use a more up to date version of Go and the SDK?

Reply:
I picked the oldest version of the SDK that supported RootDir